student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(id, age, classroom, parent_permission: true, name: 'Unknown')
+    super(id, age, parent_permission: parent_permission, name: name)
```

Comment:
- [ ] I will also suggest that the `id` parameter be removed from the `initialize` method in the Student class as well. This is because the `id` attribute is already defined and assigned within the `Person` class. As a result, when you call `super(age, name, parent_permission: parent_permission)` in the initialize method of the `Student` class, it invokes the constructor of the parent class (`Person`), which takes care of setting the `id` attribute. This way, the `id` value is passed correctly from the` parent class constructor` to the `Student` object. :wink: 

- Example Implementation
```ruby
class Student < Person 
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
  end
end
```